src/app/components/error/index.html

```diff
@@ -1,13 +1,196 @@
-<sky-demo-page
+<sky-tabbed-demo-page
   pageTitle="Error"
-  skyuxModule="SkyErrorModule"
-  packageName="@skyux/errors"
 >
 
-  <sky-demo-page-summary>
+  <sky-demo-page-summary class="sky-margin-stacked-content-section">
     The error component displays a SKY UX-themed error message. It includes options to specify an image, title, description, and action in the error message.
   </sky-demo-page-summary>
 
+  <sky-tabset id="content-tabs">
+    
+    <sky-tab tabHeading="Design">
+      
+    <sky-demo-page-content sectionHeading="Usage">
+        
+      <h3 class="sky-text-success">Use when</h3>
+        
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>Use an error to communicate system status to users in circumstances when the system encounters an unexpected error.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/error/error-usage-1.png" />
+                <div class="caption caption-success sky-text-success"> Do use an error when a system error occurs.</div>
+            </div>
+          </sky-docs-visual-example-image>
+                
+        </sky-docs-visual-example>
+        
+        <h3 class="sky-text-danger">Don't use when</h3>
+        
+        <sky-docs-visual-example>
+            
+            <sky-docs-visual-example-text>
+              <p>Don’t use an error when there is an error with a background process. Information about background processes should use <a routerLink="/components/toast">toast</a> instead.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div class="sky-margin-stacked-content-section">
+                  <img src="~/assets/img/guidelines/error/error-usage-2.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't use error for failed background processes.</div>
+                </div>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-usage-7.png" />
+                    <div class="caption caption-success sky-text-success">Do use a toast to inform the user about a process operating in the background.</div>
+                </div>
+            </sky-docs-visual-example-image>
+
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+            <p>Don’t use an error when the status of specific content needs user attention, but the system has not necessarily broken. To notify the user of a broken status on a workflow or record, use an <a routerLink="/components/alert">alert</a> instead.</p>
```

Comment:
I'd change "an error" to "error messages." Also, it's probably just me, but I'd change "needs" to "requires." ... Also, is "the system has not necessarily broken" correct? I mean, it's not the "system" that breaks is it? And "not necessarily" is kinda ambiguous. I wonder if we can change the second half of this sentence to "... user attention but no error has occurred." (I'd like to tweak that to use active voice, but that's the best I've been able to do so far.

In the second sentence, I'd change "the user" to "users." And I'd change "an alert" to "alerts."

Reply:
How about
"Don’t use error messages when the status of specific content requires user attention, but there is no system error. To notify users of a broken status on a workflow or record, use an <a routerLink="/components/alert">alerts</a> instead."

Reply:
Looks good to me.